src/utils.ts

```diff
@@ -105,3 +124,15 @@ export async function padQuerySpace(query: IServiceQuery, otherFilters?: IServic
     }
     return query;
 }
+
+export function parseRawDictData(data: string): unknown {
+    const result: any = {};
+    _.each(_.compact(_.split(data, '\n')), item => {
+        item = _.replace(_.trim(item), /^['"]|['"]$/g, '');
+        const sep = _.indexOf(item, ':');
+        if (sep > -1) {
+            result[`${_.toLower(_.trim(_.join(_.slice(item, 0, sep), '')))}`] = _.trim(_.join(_.slice(item, sep + 1), ''));
```

Comment:
not readable, may be you should create key and value consts ?

Reply:
done